ui/components/component-library/button/button.stories.tsx

```diff
@@ -192,34 +150,38 @@ export const Block = (args) => (
   </>
 );
 
-export const As = (args) => (
-  <Box display={DISPLAY.FLEX} flexDirection={FLEX_DIRECTION.ROW} gap={2}>
+export const As: StoryFn<typeof Button> = (args) => (
```

Comment:
This is an existing issue but noticed on hover the href button turns black


https://github.com/MetaMask/metamask-extension/assets/8112138/43d0d1c2-d671-40e7-862d-4817d1855396